src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
```

Comment:
add here random and colorsupplier as class fields, don't forget acess modifiers, they are also final